frontend/src/components/Header/Header.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import FoxticketLogo from '../../assets/images/logos/Foxticket.png'
+import Logout from './Logout';
+import { Link } from "react-router-dom";
+import { connect } from 'react-redux';
+import './Header.styles.sass'
+
+function Header(props) {
+  return (
+    <nav className="NavbarItems" >
+      <img src={FoxticketLogo} alt='navigation bar logo'></img>
+      <h3 className="nav-text">Foxticket</h3>
+      <ul className="nav-menu">
+      { 
+        props.type ==='admin' &&
+         <Link to="/admin" className="nav-links" >Admin</Link>
+      }
+        <li className="nav-links">{props.userName}</li>
```

Comment:
@tamas-w let's destructure props, so you don't have to write `props.*` every time